chia/rpc/wallet_rpc_api.py

```diff
@@ -864,7 +904,7 @@ async def did_get_information_needed_for_recovery(self, request):
 
     async def did_create_backup_file(self, request):
         try:
-            wallet_id = int(request["wallet_id"])
+            wallet_id = int(request["wallet_id"]) # ??? can be move out? seperate exception handling?
             did_wallet: DIDWallet = self.service.wallet_state_manager.wallets[wallet_id]
             did_wallet.create_backup(request["filename"])
             return {"wallet_id": wallet_id, "success": True}
```

Comment:
I've seen a few cases where a general `except Exception:` is catching a whole block of potential failures, like here. 
What can be worked off upfront, I'd treat explicitly (the generic Exception can still wrap it all).
But especially here the failure case looks odd since it seems confident that `wallet_id` made it (not KeyError issue). If that's just assumed to be fine (it is in functions below), then the term could be pulled out of the exception.